File: app/src/main/java/com/example/eduempoweryd/login/loginpage/LoginPageVM.java
package com.example.eduempoweryd.login.loginpage;

import android.os.Bundle;

import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class LoginPageVM extends ViewModel {
    public MutableLiveData<LoginPageModel> pageOneModel = new MutableLiveData<>(new LoginPageModel());
    private MutableLiveData<Bundle> navArguments = new MutableLiveData<>();

//    public MutableLiveData<LoginPageModel> getLoginPageModel() {
//        return LoginPageModel;
//    }

    public MutableLiveData<Bundle> getNavArguments() {
        return navArguments;
    }

    public void setNavArguments(Bundle navArguments) {
        this.navArguments.setValue(navArguments);
    }

}

File: jia_hui_s_application/app/src/main/java/com/jiahuisapplication/app/modules/loginpage/data/viewmodel/LoginPageVM.kt
package com.jiahuisapplication.app.modules.loginpage.`data`.viewmodel

import android.os.Bundle
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel
import com.jiahuisapplication.app.modules.loginpage.`data`.model.LoginPageModel
import com.jiahuisapplication.app.modules.loginpage.`data`.model.SpinnerGroupTwoModel
import kotlin.collections.MutableList
import org.koin.core.KoinComponent

class LoginPageVM : ViewModel(), KoinComponent {
  val loginPageModel: MutableLiveData<LoginPageModel> = MutableLiveData(LoginPageModel())

  var navArguments: Bundle? = null

  val spinnerGroupTwoList: MutableLiveData<MutableList<SpinnerGroupTwoModel>> = MutableLiveData()
}
